tests/trainer/test_ddp.py

```diff
@@ -143,44 +105,67 @@ def test_ddp(device: str, world_size: int, dummy_model_hparams: ModelHparams, de
     and 2) each ddp process is indeed getting different data.
     """
 
-    dummy_model_hparams.num_classes = 100
-    model = dummy_model_hparams.initialize_object()
-    assert isinstance(model, SimpleModel)
+    model = SimpleModel(num_classes=100)
```

Comment:
Remove dummy_model_hparams from arguments of test_ddp() now that we're using SimpleModel